dispatcher/MongoConnect.py

```diff
@@ -763,35 +784,35 @@ def get_run_start(self, number):
             return self.run_start_cache[str(number)]
         return None
 
-    def insert_run_doc(self, detector):
+    def insert_run_doc(self, logical):
 
         if (number := self.get_next_run_number()) == NO_NEW_RUN:
             self.logger.error("DB having a moment")
             return -1
         # the rundoc gets the physical detectors, not the logical
         detectors = self.latest_status[detector]['detectors']
```

Comment:
I think it's going to complain in this line about `detector`. Should it be `logical`?